src/game/camera.c

```diff
@@ -1148,13 +1239,18 @@ void mode_8_directions_camera(struct Camera *c) {
         s8DirModeYawOffset = snap_to_45_degrees(s8DirModeYawOffset);
     }
 #endif
-
+#endif
     lakitu_zoom(400.f, 0x900);
     c->nextYaw = update_8_directions_camera(c, c->focus, pos);
+#ifdef REONUCAM
+    set_camera_height(c, pos[1]);
+#endif
     c->pos[0] = pos[0];
     c->pos[2] = pos[2];
     sAreaYawChange = sAreaYaw - oldAreaYaw;
+#ifndef REONUCAM
     set_camera_height(c, pos[1]);
+#endif
```

Comment:
did you mean to set this twice?

Reply:
Yes. Depending on whether reonucam is enabled or not, that line of code needs to run at a different point in the function. Hence the ifdef and ifndef

Reply:
I'm blind